snippets/alloy.cson

```diff
@@ -66,4 +66,43 @@
         'rightLabelHTML': 'if (OS_ANDROID)'
         'description': 'Alloy Android conditional statement'
         'descriptionMoreURL': 'http://docs.appcelerator.com/platform/latest/#!/guide/Alloy_Controllers-section-src-34636384_AlloyControllers-Conditionalcode'
-    
\ No newline at end of file
+
+'.text.alloyxml':
+      'ListView':
+        'prefix': 'ListView'
+        'body': """
+        	<ListView id="" defaultItemTemplate="default">
+        		<Templates>
+        			<ItemTemplate name="default"></ItemTemplate>
+        		</Templates>
+        		<ListSection>
+        			<ListItem/>
+        		</ListSection>
+        	</ListView>
```

Comment:
What do you think about using some [placeholders](https://macromates.com/manual/en/snippets#placeholders)/[tab stops](https://macromates.com/manual/en/snippets#tab_stops) in these snippets? We'd provide defaults and then folks can tab through them and edit as necessary

So for example
```suggestion
        	<ListView id="${1:listview}" defaultItemTemplate="${1:default}">
        		<Templates>
        			<ItemTemplate name="${1:default}"></ItemTemplate>
        		</Templates>
        		<ListSection>
        			<ListItem/>
        		</ListSection>
        	</ListView>
```